### ISSUE TITLE
Proposing removing the nodejs/Foundation GitHub team

### ISSUE THREAD

**Issue by Trott** [nodejs/admin#42]
Not sure why the @nodejs/Foundation team was originally created, but I think we can and should remove it.

* It contains four people, of which only one (Mark Hinkle) is actually a Foundation person.
* It has no repositories.
* It has no subteams.
* It has an empty discussion board.
* Membership supplies access to the private moderation repository. :scream: 

For whatever purpose it was set up, it does not appear to be serving that purpose as far as I can tell. Which is fine. No judgment or anything. I just think we should remove it. Thoughts @nodejs/tsc and @nodejs/community-committee?

**Comment by Trott** [comment]
Not sure why the @-mention in the original post didn't work, but let me try again: @nodejs/foundation

**Comment by mhdawson** [comment]
I think I can see the link to the foundation (past or present) for at least 3 of the 4.  It may have been to specifically to give access to the moderation repo.

I think that only 2 of the 4 would lose that access if we removed the team.

I'm ok if we remove the team, but I think a bigger question is if Foundation representatives should have access to the moderation repo  ? @nodejs/moderation what's your take on that ?

**Comment by cjihrig** [comment]
+1 to removing it.

**Comment by bnb** [comment]
For what it's worth, @zibbykeaton and @gtwallace have both been engaging more in the project on GitHub - I do think it would be helpful to enable them to participate fully by having them be members of the org in some capacity (not necessarily @nodejs/foundation).

**Comment by gtewallace** [comment]
+1 I (think :) truth be told, I don't really know what it means to "be a member" but I do love working with everyone.  And so much of what we do in the Foundation is inextricably linked with the community. I was working on an email today and wrote that the Foundation's marketing function literally exists to help broadcast helpful Node.js stories - be they about technology, technologists or enterprises. So I am all for whatever capacity facilitates us doing that better!

**Comment by benjamingr** [comment]
You can add the people removed to nodejs/members

**Comment by ZibbyKeaton** [comment]
I really like to participate on GitHub, it helps me get a better understanding who everyone is, where there is need and how our marketing function can support those needs, and it keeps me informed on any news that might need our attention (announcements, security, etc.). As @gtewallace mentioned whatever this group thinks is best to ensure we are telling Node.js stories in whatever context they may form.

**Comment by Trott** [comment]
> For what it's worth, @ZibbyKeaton and @gtewallace have both been engaging more in the project on GitHub - I do think it would be helpful to enable them to participate fully by having them be members of the org in some capacity (not necessarily @nodejs/foundation).

They already are. Zibby is on website and nodejs-collection teams, and Greg is on evangelism.

(Even if they weren't, I'm not sure what having them in the org would buy them other than access to the moderation repo.)

**Comment by ashleygwilliams** [comment]
fwiw it appears that @sarahnovotny is on that group. she is a board director and i think we should continue her access (whether via this team or another, i have zero opinion on)

**Comment by jasnell** [comment]
> It contains four people, of which only one (Mark Hinkle) is actually a Foundation person.

I created the team and at the time it was created all four were directly involved with the Foundation and had been actively participating. Since that time, the personal situation for two of the members has changed, and it would be worthwhile to remove them from the team. I would prefer to keep the team and expand it's membership to include individuals like @ZibbyKeaton, @gtewallace, and others.

**Comment by Trott** [comment]
> fwiw it appears that @sarahnovotny is on that group. she is a board director and i think we should continue her access (whether via this team or another, i have zero opinion on)

There is a nodejs/board team she could be added to.

Again, though, these teams are mostly unused. For the most part, all being on a team buys you is the ability to see the moderation repo and the ability to be @-mentioned (which doesn't happen for most teams at all).

**Comment by Trott** [comment]
> I would prefer to keep the team and expand it's membership

The cost of keeping unused teams includes:

* stymies efforts at cleanup and increased security
* makes it harder to find the *right* team (should I @-mention foundation team? board team? some other team? I mean, I know the difference and you do, but nearly nobody else does)
* since having literally dozens of teams that no one actively manages pretty much guarantees that the membership is wrong, it enhances the sense people have that the teams are inaccurate. And that encourages people to list out individuals and ignore teams, even the useful ones.

**Comment by ashleygwilliams** [comment]
@Trott i think combining board and foundation seems like a reasonable idea. the points you list here are really good and important. i dont think the foundation vs board distinction is worth preserving in the github org cuz the difference isn't functionally useful to how we conduct ourselves here.
